e2e/testdata/regressions/issue_5399.def

```diff
@@ -1,5 +1,5 @@
-bootstrap: oras
```

Comment:
@DrDaveD @ikaneshiro we need Oras E2E tests ... not sure that we want to revert all to `library://`

Reply:
This was changed back by a later commit